config/config.yaml

```diff
@@ -109,8 +116,9 @@ storage:
 
     # Max number of threads (jobs) for running optimizations per shard.
     # Note: each optimization job will also use `max_indexing_threads` threads by itself for index building.
+    # If null - have no limit and choose dynamically to saturate CPU.
     # If 0 - no optimization threads, optimizations will be disabled.
-    max_optimization_threads: 1
+    max_optimization_threads: null
```

Comment:
`optimizers.max_optimization_threads` is per shard, not per collection. Now supports `null` to have no default and dynamically saturate the CPU, this is the new default.